JSDemos/Demos/DataGrid/PDFOverview/description.md

```diff
@@ -1,8 +1,12 @@
-Our [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to easily and accurately export its contents to a PDF document. To enable PDF export operations, you must reference or import the following:
+The [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to export its contents to a PDF document. To enable PDF export operations, import the <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library.
```

Comment:
I think the second sentence deserves a separate paragraph. The first sentence introduces the entire idea of the demo - export the grid to PDF (what it's all about). The second is one of the steps to get there (something you should do).